addons/web_dashboard/static/src/js/dashboard.js

```diff
@@ -0,0 +1,256 @@
+odoo.define('dashboard.dashboard', function (require) {
+"use strict";
+
+var core = require('web.core');
+var Widget = require('web.Widget');
+var ajax = require('web.ajax');
+var Planner = require('planner.planner');
+
+var QWeb = core.qweb;
+var _t = core._t;
+
+var Dashboard = Widget.extend({
+    template: 'DashboardMain',
+    events: {
+        'click .install_new_apps': 'new_apps',
+    },
+    start:function(){
+        var self = this;
+        ajax.jsonRpc('/web/dataset/call', 'call', {
+            model: 'ir.model.data',
+            method: 'xmlid_to_res_id',
+            args: ['base.view_users_form'],
+        }).then(function(res_id) {
+            self.user_form_id = res_id;
+            self.load();
+        });
+    },
+    load:function(){
+        var self = this;
+        ajax.jsonRpc("/dashboard/info", 'call', {}).then(function (data) {
+            self.$el.find('#apps').append(QWeb.render("DashboardApps", data));
+            new DashboardInvitations(self,data).appendTo('#invitations');
+            new DashboardPlanner(self,data).appendTo('#planner');
+            new DashboardVideos(self,data).appendTo('#videos');
```

Comment:
You are passing `data` to your 3 sub-widgets, but they don't use `data`. Even the planner widget re-fetch planner data with `fetch_application_planner`. This is performance killer. Privilege one ONE rpc call to `/dashboard/info` to get ALL the information your need for your widget, and give only the required information to your widget constructor (`new DashboardInvitations(self,data['users']) .... ew DashboardVideos(self,data['planners']) ....`)